now.json

```diff
@@ -0,0 +1,13 @@
+{
+  "builds": [
+    {
+      "config": {
+        "distDir": "public"
+      },
+      "src": "package.json",
+      "use": "@now/static-build"
+    }
+  ],
+  "name": "qb-schema",
+  "version": 2
```

Comment:
Is this right? What does it represent?

Reply:
This is the version of the now.sh platform the script deploys to: https://zeit.co/docs/v2/deployments/configuration/#version